app/filter.js

```diff
@@ -1,21 +1,23 @@
-import React from 'react';
-import { SafeAreaView, Text, View, StatusBar } from 'react-native';
-import { styled } from 'nativewind';
-import { Button } from '../components/Buttons';
+import React from "react";
```

Comment:
change your formatter in settings to use single quotes, otherwise there will be a lot of these quote replacements happening. Our standard for development is single quotes